governance/program/src/processor/process_create_token_governance.rs

```diff
@@ -84,13 +86,28 @@ pub fn process_create_token_governance(
         rent,
     )?;
 
-    if transfer_token_owner {
-        set_spl_token_owner(
+    if transfer_account_authorities {
+        set_spl_token_account_authority(
             governed_token_info,
             governed_token_owner_info,
             token_governance_info.key,
+            AuthorityType::AccountOwner,
             spl_token_info,
         )?;
+
+        // If the token account has close_authority then transfer it as well
+        let token_account_data = Account::unpack(&governed_token_info.data.borrow())?;
+        // Note: The code assumes owner==close_authority
+        //       If this is not the case then the caller should set close_authority accordingly before making the transfer
+        if token_account_data.close_authority.is_some() {
+            set_spl_token_account_authority(
+                governed_token_info,
+                governed_token_owner_info,
+                token_governance_info.key,
+                AuthorityType::CloseAccount,
+                spl_token_info,
+            )?;
+        }
```

Comment:
As mentioned earlier, I can't see a use-case for this, so you can probably remove it

Reply:
This is to ensure all the authorities are transferred. If the owner can't also have `CloseAuthority` if it's set then the transfer shouldn't be possible.